pkg/libvirttools/virtualization.go

```diff
@@ -277,6 +280,58 @@ func (v *VirtualizationTool) addVolumesToDomain(podID, podName string, domain *l
 	return nil
 }
 
+func vmLogToFile() bool {
+	if os.Getenv("VIRTLET_VM_LOG_TO_FILE") != "" {
+		return true
+	}
+	return false
+}
+
+func (v *VirtualizationTool) RemoveLibvirtSandboxLog(sandboxId string) error {
+	return os.RemoveAll(filepath.Join(vmLogBaseDir, sandboxId))
+}
+
+func (v *VirtualizationTool) addSerialDevicesToDomain(sandboxId string, domain *libvirtxml.Domain, settings VirtletDomainSettings) error {
+	if settings.vmLogToFile {
+		logDir := filepath.Join(vmLogBaseDir, sandboxId)
+		logPath := filepath.Join(logDir, vmLogFileName)
+
+		// Prepare directory where libvirt will store log file to.
+		if err := os.Mkdir(logDir, 0644); err != nil {
+			glog.Warningf("failed to create vmLogDir '%s': %s", logDir, err.Error())
```

Comment:
Shouldn't we return an error there on `Mkdir` failure?
Passing that there means that we are postponing an error to qemu invocation, which then should fail on write to file.
Also this warning can be seen only in Virtlet logs, so there will be no solid info about it during pod execution in pod events/logs.